sdk/src/main/java/io/radar/sdk/RadarState.kt

```diff
@@ -179,4 +183,36 @@ internal object RadarState {
         }
     }
 
+    internal fun getGeofenceIds(context: Context): MutableSet<String>? {
+        return getSharedPreferences(context).getStringSet(KEY_GEOFENCE_IDS, null)
+    }
+
+    internal fun setGeofenceIds(context: Context, geofenceIds: Set<String>?) {
+        getSharedPreferences(context).edit { putStringSet(KEY_GEOFENCE_IDS, geofenceIds) }
```

Comment:
There's no `putArray()` on `SharedPreferences`, so we need to use `putStringSet()` to store these instead, and call `toTypedArray()` when sending to `POST /track`